src/themes/hugo-theme-learn/static/css/theme.css

```diff
@@ -631,8 +631,8 @@ code {
     border-radius: 2px;
     white-space: nowrap;
     color: #5e5e5e;
-    background: #FFF7DD;
-    border: 1px solid #fbf0cb;
+    background: #f2f2f2;
+    border: 1px solid #eeeeee;
```

Comment:
It's funny, the border on these things is so subtle it's barely noticeable (And I'm on 27" monitor at 1080p :P)

Reply:
Yeah it’s not intended to be noticed I think , we could probably ditch it out, if you make it visible it’s harder to read.